README.md

```diff
@@ -96,35 +106,35 @@ By the end of this lesson. You should be able to set up two separate servers tha
 
 * **Question:** What is happening on the first couple of lines of the new file you've created?
 
-* **Your Answer:** 
+* **Your Answer: sets environmental variables allowing us to use a local host when in a deve environment and then the ability to set the actual host once we are in a production environment
 
 ---
 
 - [ ] Let's store the token in LocalStorage with a key of `journal-app`.
 
 * **Question:** Why are we storing the token?
 
-* **Your Answer:**
+* **Your Answer: so it can be used by the application to authorized access or other needs
```

Comment:
Yes, if we do not store it we will need to re log in everytime a user refreshes or manually visits a route